src/modules/emote_menu/components/Menu.jsx

```diff
@@ -90,7 +95,9 @@ export default function EmoteMenu({triggerRef, appendToChat}) {
 
   return (
     <>
-      <Header className={styles.header} value={search} onChange={setSearch} onHide={onHide} selected={selected} />
+      <TipWhisper>
```

Comment:
rather than making a tooltip, i'd prefer to designate a part of the menu container to showing tips. We could allocate a small 16px box at the bottom to show tips in, if available.